sql/localsql/schema/migrations/0010_atxs.sql

```diff
@@ -0,0 +1,11 @@
+--- Table for storing blobs of published ATX for regossiping purposes.
+CREATE TABLE atx_blobs
+(
+    id         CHAR(32) PRIMARY KEY,
+    pubkey     CHAR(32) NOT NULL,
+    epoch      INT NOT NULL,
+    atx        BLOB,
+    version    INTEGER
+);
+
+CREATE UNIQUE INDEX atx_blobs_epoch_pubkey ON atx_blobs (epoch, pubkey);
```

Comment:
I'm not sure I understand why we need this? Isn't the node re-gossiping automatically? Also when a peer requests all ATXs for an epoch the self published ATXs will be contained?